### DISCUSSION TITLE
How to add customer order note?

### DISCUSSION THREAD

**Discussion by gtomaka** [saleor/saleor#8769] [General]
Hi! Thanks for creating such amazing product. I have one question regarding Saleor.
According to your docs regarding checkout (https://docs.saleor.io/docs/3.0/developer/checkout) we are suppose to perform process flow with completeCheckout mutation that will create an order. We would like to let user type some note regarding his order. How can we achieve that?

Cheers, Grzegorz.

**Comment by karolkielecki** [comment]
@grzesiektomaka Our [SDK](https://www.npmjs.com/package/@saleor/sdk/v/next) doesn't support checkout for now. Currently, there is no official solution.

**Comment by krzysztofwolski** [comment]
Note can also be added as metadata to checkout - during the order creation metadata are copied.
